frontend/src/components/basic-detail/BasicDetail.tsx

```diff
@@ -58,42 +53,36 @@ const BasicDetail: React.FC<BasicDetailProp> = (props: BasicDetailProp) => {
         .then((data) => setResourcePath(data));
     // eslint-disable-next-line
   }, []);
-  const Myfun=(it:any) =>{
+  const ClipboardItem=(it:any) =>{
     return (
-
-      <Flex breakpointMods={[{modifier: 'row', breakpoint: 'lg'}]}>
-        <FlexItem>
-
-          <ClipboardCopy style = {{width: '53em'}}
-            isReadOnly variant={ClipboardCopyVariant.expansion}>
-            {`kubectl apply -f ${it.it}`}</ClipboardCopy>
-          <br />
-        </FlexItem>
-      </Flex>
+      <React.Fragment>
+        <ClipboardCopy style={{marginBottom: '2em'}}
+          isReadOnly variant={ClipboardCopyVariant.expansion}>
+          {`kubectl apply -f ${it.item}`}</ClipboardCopy>
+      </React.Fragment>
```

Comment:
where does the `Flex` get closed?

Reply:
I have used Flex in return which is closing at line no.  193

Reply:
> where does the `Flex` get closed?

I have used Flex in return which is closing at line no 193